From b9dd2241c60635ba15929ffad9962928460bd3ec Mon Sep 17 00:00:00 2001
From: Miklos Vajna <vmiklos@frugalware.org>
Date: Tue, 3 May 2011 14:27:06 +0200
Subject: [PATCH] ooinstall: respect DESTDIR

The problem was that ooinstall just passed -destdir to
make_installer.pl, but before that, it tried to create the root
directory, where it did not respect DESTDIR.

Signed-off-by: Petr Mladek <pmladek@suse.cz>
---
 solenv/bin/ooinstall |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/solenv/bin/ooinstall b/solenv/bin/ooinstall
index 83717fa..47c06cb 100755
--- a/solenv/bin/ooinstall
+++ b/solenv/bin/ooinstall
@@ -29,7 +29,7 @@ for $arg (@ARGV) {
 	$help = 1;
     } else {
 	# Cwd::realpath does not work if the path does not exist
-	mkpath($arg) unless -d $arg;
+	mkpath($ENV{DESTDIR} . $arg) unless -d $ENV{DESTDIR} . $arg;
 	$path = Cwd::realpath( $arg );
     }
 }
-- 
1.7.4.4

